<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordEditDialog</class>
 <widget class="QDialog" name="RecordEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>1021</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutMain">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="webViewRecord"/>
      <widget class="QTextEdit" name="textEditRecord"/>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="tabData">
        <attribute name="title">
         <string>Данные</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Дата:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditDate"/>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Тип:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxType">
             <property name="editable">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Метрическая книга - запись о рождении</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Метрическая книга - запись о браке</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Исповедная роспись</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ревизская сказка</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Переписная книга</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Источник:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="lineEditSource">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonSearchSource">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Местонахождение в источнике:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditLocation"/>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Место, где сделана запись:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLineEdit" name="lineEditPlace">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonRecordPlaceSearch">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Заметки:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEditNote"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabHumans">
        <attribute name="title">
         <string>Люди</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>321</width>
           <height>311</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutHumans">
          <item>
           <widget class="QListWidget" name="listWidgetHumans"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButtonNewHuman">
              <property name="text">
               <string>Новый</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSearchHuman">
              <property name="text">
               <string>Искать</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonRemoveHuman">
              <property name="text">
               <string>Убрать</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tabPlaces">
        <attribute name="title">
         <string>Места</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>311</width>
           <height>321</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutPlaces">
          <item>
           <widget class="QListWidget" name="listWidgetPlaces"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButtonNewPlace">
              <property name="text">
               <string>Новое</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSearchPlace">
              <property name="text">
               <string>Искать</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonRemovePlace">
              <property name="text">
               <string>Убрать</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tabSources">
        <attribute name="title">
         <string>Источники</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>40</y>
           <width>221</width>
           <height>191</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutSources">
          <item>
           <widget class="QListWidget" name="listWidgetSources"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonNewRecordSource">
              <property name="text">
               <string>Новый</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonDeleteRecordSource">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButtonEdit">
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonApply">
        <property name="text">
         <string>Посмотреть</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPlusFont">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonMinusFont">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="text">
         <string>Удалить запись</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOk">
        <property name="text">
         <string>Добавить и закрыть</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditDate</tabstop>
  <tabstop>comboBoxType</tabstop>
  <tabstop>pushButtonSearchSource</tabstop>
  <tabstop>pushButtonRecordPlaceSearch</tabstop>
  <tabstop>textEditNote</tabstop>
  <tabstop>pushButtonNewHuman</tabstop>
  <tabstop>pushButtonSearchHuman</tabstop>
  <tabstop>pushButtonRemoveHuman</tabstop>
  <tabstop>pushButtonNewPlace</tabstop>
  <tabstop>pushButtonSearchPlace</tabstop>
  <tabstop>pushButtonRemovePlace</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>listWidgetHumans</tabstop>
  <tabstop>listWidgetPlaces</tabstop>
  <tabstop>textEditRecord</tabstop>
  <tabstop>lineEditSource</tabstop>
  <tabstop>lineEditPlace</tabstop>
  <tabstop>lineEditLocation</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
